File: terra/src/main/scala/com/dafttech/terra/game/world/subtiles/SubtileBone.java
package com.dafttech.terra.game.world.subtiles;

import com.badlogic.gdx.graphics.g2d.TextureRegion;
import com.dafttech.terra.resources.Resources$;

public class SubtileBone extends Subtile {
    public SubtileBone() {
        super();
    }

    @Override
    public TextureRegion getImage() {
        return Resources$.MODULE$.TILES().getImage("bone");
    }

}

File: terra/src/main/scala/com/dafttech/terra/game/world/subtiles/SubtileBone.scala
package com.dafttech.terra.game.world.subtiles

import com.badlogic.gdx.graphics.g2d.TextureRegion
import com.dafttech.terra.resources.Resources
import monix.eval.Task

class SubtileBone extends Subtile {
  override def getImage: Task[TextureRegion] = Resources.TILES.getImage("bone")
}